equinity-historical-data/src/test/resources/Crm14MockFile_5001669.txt

```diff
@@ -1650,6 +1650,35 @@
                 "Business_trading_date": "2015-01-01T00:00:00"
             }
         },
+        "Partner_business_details": {
+            "row": {
```

Comment:
Next time, it is better to add a new file to cover this case. Because this file could be used for other specific tests.
We also need more examples with a list  of 1 and a list of many. And update tests accordingly